src/platform/packages/shared/shared-ux/page/solution_nav/src/collapse_button.scss

```diff
@@ -1,49 +0,0 @@
-@import 'variables';
-
-.kbnSolutionNavCollapseButton {
-  position: absolute;
-  opacity: 0;
-  left: $solutionNavWidth - $euiSize;
-  top: $euiSizeL;
-  z-index: 2;
-
-  @include euiCanAnimate {
-    transition: opacity $euiAnimSpeedFast, left $euiAnimSpeedFast, background $euiAnimSpeedFast;
-  }
-
-  &:hover,
-  &:focus {
-    transition-delay: 0s !important;
-  }
-
-  .kbnSolutionNav__sidebar:hover &,
-  &:hover,
-  &:focus {
-    opacity: 1;
-    left: $solutionNavWidth - $euiSizeL;
-  }
-
-  .kbnSolutionNav__sidebar:hover & {
-    transition-delay: $euiAnimSpeedSlow * 2;
-  }
-
-  &:not(&-isCollapsed) {
-    background-color: $euiColorEmptyShade  !important; // Override all states
-  }
```

Comment:
`&-isCollapsed` no longer represents a valid class name, so I redid this logic in JS by adding this style to an object named `styles.noCollapsed`.